.circleci/config.yml

```diff
@@ -19,12 +116,34 @@ workflows:
                 - latest
                 - lts
                 - maintenance
+      - external-nut:
+          requires:
+            - release-management/test-package
+          sfdx_version: latest
+          matrix:
+            parameters:
+              os: [linux]
```

Comment:
@mshanemc is Windows going to be added once external-nuts is stable on Linux?

Reply:
I'd like to do that separately.  We have a ticket open with Circle to get newer windows images because
1) they're node 12 by default
2) we've run into several weird errors installing other versions of node